python/gym-unity/setup.py

```diff
@@ -10,5 +10,5 @@
       author_email='ML-Agents@unity3d.com',
       url='https://github.com/Unity-Technologies/ml-agents',
       packages=find_packages(),
-      install_requires = ['gym', 'unityagents']
```

Comment:
Technically this only needs to be 'mlagents[envs]`.

Reply:
To be clear, it doesn't require `gym`?

Reply:
It does! Just not `mlagents.trainers`.